extensions/2.0/Vendor/EXT_feature_metadata/schema/featureCollection.schema.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "Feature collection",
+    "type": "object",
+    "description": "A feature collection designated by a feature class and property values stored in arrays.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
```

Comment:
We should include a `name` field here and make the container for this a dictionary instead of an array.

This helps with both consistency and makes it easier to understand references to the collection.